libs/blocks/global-navigation/utilities/getUserEventHistory.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-promise-executor-return, no-async-promise-executor */
+import { getConfig } from '../../../utils/utils.js';
+
+let userEventHistory;
+const getUserEventHistory = async (id) => {
+  if (userEventHistory) return userEventHistory;
+  if (!window.adobeIMS?.isSignedInUser()) return Promise.reject(new Error('User not signed in'));
+  let resolve;
+  let reject;
+  userEventHistory = userEventHistory || new Promise((_resolve, _reject) => {
+    resolve = _resolve;
+    reject = _reject;
+  });
+  const profile = await window.adobeIMS.getProfile();
+  const apiUrl = getConfig().env.name === 'prod'
+    ? 'https://www.adobe.com/gating/api/realtime-profile'
+    : 'https://www.stage.adobe.com/gating/api/realtime-profile';
+  const res = await fetch(`${apiUrl}?guid=${id || profile.userId}`, {
+    method: 'GET',
```

Comment:
you don't need to pass the method to the fetch, as 'GET' is the default
I actually don't even think that you need to pass 'Content-Type' or Accept either. It should just work, so might as well save on those bytes :)

Reply:
https://github.com/adobecom/milo/pull/651